src/components/CoffeeMatch/CoffeeMatch.jsx

```diff
@@ -20,20 +17,42 @@ const CoffeeMatch = ({ answers }) => {
     [acidity, characteristics, profile, roastLevel]
   );
 
+  const primaryMatch = matches[0]; // primary match is the first item in the array
+  const otherMatches = matches.slice(1, 4); // up to 3 additional matches
+
   return (
     <div className="match">
-      <h3 className="coffee-match-title">Here is your Coffee match!</h3>
+      <h3 className="coffee-match-title">Here are your Coffee matches!</h3>
```

Comment:
We could do a ternary here:
```suggestion
      <h3 className="coffee-match-title">{matches.length > 0 ? "Here are your Coffee matches!" : "Here is your Coffee match!"</h3>
```